layout/includes/slide/script.pug

```diff
@@ -25,7 +28,11 @@ script.
     parallaxBackgroundVertical: '!{parallaxBackgroundVertical}',
     autoSlide: !{autoSlide},
     loop: !{loop},
-    autoSlideStoppable: !{autoSlideStoppable},
+    controlsLayout: '!{controlsLayout}', // Determines where controls appear, "edges" or "bottom-right"
+    controlsBackArrows: '!{controlsBackArrows}',
+    //- progress: !{progress},
```

Comment:
why comment these lines

Reply:
L33 - L35

Reply:
Fixed below, all of them are uncommented